src/packageurl/contrib/purl2url.py

```diff
@@ -24,6 +24,7 @@
 # Visit https://github.com/package-url/packageurl-python for support and
 # download.
 
+
```

Comment:
Remove this

Reply:
That comment also appears at the top of `/packageurl-python/tests/contrib/test_purl2url.py` -- I've deleted it there as well.

Reply:
I meant to remove the empty line.